doc/WebRTC.xml

```diff
@@ -1,1029 +1,653 @@
+                <para>STUN</para>
+              </entry>
+              <entry valign="middle">
+                <para>Session Traversal Utilities for NAT</para>
+              </entry>
+            </row>
+            <row>
+              <entry valign="middle">
+                <para>TURN</para>
+              </entry>
+              <entry valign="middle">
+                <para>Traversal Using Relays around NAT</para>
+              </entry>
+            </row>
+            <row>
+              <entry valign="middle">
+                <para>WebRTC</para>
+              </entry>
+              <entry valign="middle">
+                <para>Web Real-Time Communication</para>
+              </entry>
+            </row>
+          </tbody>
+        </tgroup>
+      </informaltable>
+    </section>
+  </chapter>
+  <chapter xml:id="chapter_yxt_v12_v5b">
+    <title>Overview</title>
+    <para>The WebRTC standard includes APIs for communicating with an ICE Server, but the signaling
+      component is not part of it. Signaling is needed in order for two peers to share how they
+      should connect.</para>
+    <para>Signaling can be implemented in many different ways, and the WebRTC standard doesn't
+      prefer any specific solution.</para>
+    <para>This specification contains documentation and examples of the signaling protocol used in
+      ONVIF to set up a WebRTC peer-to-peer connection. The setup involves three participants:
+        <emphasis role="italic">client</emphasis>, <emphasis role="italic">device</emphasis> and
+        <emphasis role="italic">signaling server</emphasis>.</para>
+    <figure xml:id="_Ref493258796">
+      <title>Client, device, signaling server</title>
+      <mediaobject>
+        <imageobject>
+          <imagedata fileref="media/WebRTC/client_device_server_triangle.svg"
+            contentwidth="120mm"/>
+        </imageobject>
+      </mediaobject>
+    </figure>
+    <para>In this figure,<itemizedlist>
+        <listitem>
+          <para>The <emphasis role="italic">client</emphasis> represents a real user who initiates
+            the WebRTC session.</para>
+        </listitem>
+        <listitem>
+          <para>The <emphasis role="italic">device</emphasis> is the resource delivering the media,
+            for example a camera.</para>
+        </listitem>
+        <listitem>
+          <para>The <emphasis role="italic">signaling server</emphasis> is the mediator that both
+              <emphasis role="italic">client</emphasis> and <emphasis role="italic"
+              >device</emphasis> connect to in order to establish a peer-to-peer connection between
+            them.</para>
+        </listitem>
+      </itemizedlist></para>
+    <para>The <emphasis>Signaling Protocol</emphasis> described in this specification details the
+      data exchange between client and device via the signaling server. The formal API is defined in
+      a separate AsyncAPI schema file.</para>

```

Comment:
We are no more referring to "Async API schema" files and hence specification must remove referring to them?